common-app-lib/src/cmr/common_app/api/request_logger.clj

```diff
@@ -61,6 +65,26 @@
        ;; Parameters in forms could be really large, don't log everything
        (as-> item (apply dissoc item (drop limit (keys item)))))))
 
+(defn- normalize-local
+  "IP 6 addresses are ugly and wildly different then IP 4, normalize these so that all localhost
+   searches can be found in logs regardless of technology used to deliver the request."
+  [raw-address]
+  (if (contains? (set ["[0:0:0:0:0:0:0:1]" "127.0.0.1"]) raw-address)
+    "localhost"
+    raw-address))
```

Comment:
if a system calls itself through transmit, or if running locally this comes into play. I also just felt that IP6 addresser were ugly.